src/renderer/component/publishForm/view.jsx

```diff
@@ -1,884 +1,574 @@
+                  <option value="de">{__('German')}</option>
+                  <option value="jp">{__('Japanese')}</option>
+                  <option value="ru">{__('Russian')}</option>
+                  <option value="es">{__('Spanish')}</option>
+                </FormField>
+              </FormRow>
+
+              <LicenseType
+                licenseType={licenseType}
+                otherLicenseDescription={otherLicenseDescription}
+                licenseUrl={licenseUrl}
+                copyrightNotice={copyrightNotice}
+                handleLicenseChange={(licenseType, licenseUrl) => updatePublishForm({ licenseType, licenseUrl })}
+                handleLicenseDescriptionChange={(event) => updatePublishForm({ otherLicenseDescription: event.target.value })}
+                handleLicenseUrlChange={(event) => updatePublishForm({ licenseUrl: event.target.value })}
+                handleCopyrightNoticeChange={(event) => updatePublishForm({ copyrightNotice: event.target.value })}
+              />
+            </section>
+
+            <section className="card card--section">
+              <div className="card__title">
+                {__('Terms of Service')}
+              </div>
+              <div className="card__content">
+                <FormField
+                  name="lbry_tos"
+                  type="checkbox"
+                  checked={tosAccepted}
+                  postfix={
+                    <span>
+                      {__('I agree to the')}{' '}
+                      <Button
+                        fakeLink
+                        href="https://www.lbry.io/termsofservice"
+                        label={__('LBRY terms of service')}
+                      />
+                    </span>
+                  }
+                  onChange={event => updatePublishForm({ tosAccepted: event.target.checked })}
+                />
+              </div>
+            </section>
+
+            <div className="card__actions">
+              <Submit
+                label={!publishing ? __('Publish') : __('Publishing...')}
+                disabled={
+                  formDisabled ||
+                  !formValid ||
+                  publishing
+                }/>
+                <Button alt onClick={this.handleCancelPublish} label={__('Cancel')} />
+            </div>
+            {!formDisabled && !formValid && this.renderFormErrors()}
+          </div>
+        </Form>
+        <Modal
+          isOpen={publishSuccess}
+          contentLabel={__('File published')}
+          onConfirmed={() => {
+            clearPublish();
+            navigate('/published');
+          }}
+        >
+          <p>
+            {__('Your file has been published to LBRY at the address')}{' '}
+            <code>{uri}</code>!
+          </p>
+          <p>
+            {__(
+              'The file will take a few minutes to appear for other LBRY users. Until then it will be listed as "pending" under your published files.'
```

Comment:
Probably out of scope, but this is something we could allow a user to acknowledge and not see again.